main.go

```diff
@@ -0,0 +1,31 @@
+//Package main is the entry point of the API. It takes care of setting up and starting the server.
+package main
+
+import (
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/daivikd3v/User-API/handlers"
+	muxhandlers "github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+)
+
+func main() {
+	router := mux.NewRouter()
+
+	handlers.RegisterRoutes(router)
```

Comment:
All code from `main()` function can be written in `handlers.RegisterRoutes()` function.